src/components/playerstats/playerstats.js

```diff
@@ -274,17 +274,55 @@ function getMediaSourceStats(session, player) {
         });
     }
 
+    const frameRate = videoStream.AverageFrameRate || videoStream.RealFrameRate;
+    if (frameRate) {
+        sessionStats.push({
+            label: globalize.translate('Framerate'),
+            value: frameRate
+        });
+    }
+
     if (videoStream.BitRate) {
         sessionStats.push({
             label: globalize.translate('LabelVideoBitrate'),
             value: getDisplayBitrate(videoStream.BitRate)
         });
     }
 
+    if (videoStream.BitDepth) {
+        sessionStats.push({
+            label: globalize.translate('MediaInfoBitDepth'),
+            value: `${videoStream.BitDepth} bit`
+        });
+    }
+
     if (videoStream.VideoRangeType) {
         sessionStats.push({
             label: globalize.translate('LabelVideoRangeType'),
-            value: videoStream.VideoDoViTitle || videoStream.VideoRangeType
+            value: videoStream.VideoDoViTitle ?
+                `${videoStream.VideoDoViTitle} [${videoStream.VideoRangeType}]` :
+                videoStream.VideoRangeType
+        });
```

Comment:
It would be better to list `VideoDoViTitle` on a separate line and add `translate('LabelVideoDoViTitle')`